net/golioth/settings.c

```diff
@@ -95,6 +96,27 @@ static int send_status_report(struct golioth_client *client,
 	return send_coap_response(client, response_buf, nbytes);
 }
 
+static void save_setting(const char *key, const struct golioth_settings_value *value)
+{
+	int err;
+	char name[64] = {};
```

Comment:
Shouldn't we use something like `sizeof("gsettings/") + GOLIOTH_SETTINGS_MAX_NAME_LEN` instead of `64`?